app/views/reality_check/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="container container-custom">
+  <h1 class="oompr-title"><%= link_to 'oompr', root_path, class: 'oompr-title-link' %></h1>
+  <div class="row">
+    <div class="col-md-6 offset-md-3">
+      <div class="listings-container">
+        <% if @rc_listings.empty? %>
+          <div class="alert alert-warning" role="alert">
+            No listings found. Want to search for rentals in your area?
+          </div>
```

Comment:
thanks for including this rental prompt - not sure if we'll have the chance to implement it as part of our mvp, but thank you for being so detail oriented